introduction.md

```diff
@@ -1,3 +1,6 @@
 hello i'm david i'm working on code school courses yeah!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?